gui/src/app/facilities/facilities-detail/facilities-detail.component.html

```diff
@@ -32,13 +32,20 @@ <h2 class="card-title">
       <h4 class="mt-4">{{'FACILITIES.FACILITY_ATTRIBUTES' | translate}}</h4>
 
       <table mat-table [dataSource]="facilityAttributes" class="w-100">
-        <ng-container matColumnDef="fullname">
+        <ng-container matColumnDef="fullname" class="mr-5">
           <th mat-header-cell *matHeaderCellDef>{{'FACILITIES.FACILITY_DETAIL_ITEM_NAME' | translate}}</th>
-          <td mat-cell *matCellDef="let attribute">{{attribute.fullName}}</td>
+          <td mat-cell *matCellDef="let attribute" class="font-weight-bold">{{attribute.fullName}}</td>
```

Comment:
This weight is too high for current font and font size, it looks blurry. Try to find a class with `font-weight: 500 !important;` or create one yourself.